pkgs/by-name/in/inori/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "inori";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "eshrh";
+    repo = "inori";
+    rev = "v${version}";
+    hash = "sha256-yd1kIlPepVbeTEFzjxTDWEh8o4m6dh/ya9GitqYHHT8=";
+  };
+
+  cargoHash = "sha256-mmzJXIl0wfcyEaYc2wZfA22ZEbTQXG5LVqxEQ4+x76M=";
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Client for the Music Player Daemon (MPD)";
+    homepage = "https://github.com/eshrh/inori";
+    license = lib.licenses.gpl3Only;
+    platforms = lib.platforms.all;
```

Comment:
```suggestion
    platforms = lib.platforms.unix;
```

Reply:
MPD [works on Windows](https://www.musicpd.org/download.html) (although nixpkgs doesn't package it), and inori tries to be platform-independent by using crates like [dirs](https://docs.rs/dirs/latest/dirs/). As far as I know, inori should work on Windows (although this is untested).

https://github.com/NixOS/nixpkgs/blob/8f3e1f807051e32d8c95cd12b9b421623850a34d/pkgs/servers/mpd/default.nix#L286